packages/vector_graphics_compiler/pubspec.yaml

```diff
@@ -36,7 +36,7 @@ dev_dependencies:
     sdk: flutter
   test: ^1.20.1
   # See https://github.com/flutter/flutter/issues/157626
-  vector_graphics: ">=1.1.11+1 <= 1.1.12"
+  vector_graphics: ">=1.1.11+1 <= 1.1.14"
```

Comment:
Isn't 11.1.13 the minimum required for wasm compatibility?

Reply:
vector_graphics was already at 1.1.13, I bumped it to 1.1.14 but the v1.1.13 was not used here, no idea why

Reply:
You bumped the maximum, not the minimum. The CHANGELOG entry says this PR makes the package wasm compatible, which would generally involve also setting the minimum versions of any dependencies to wasm-compatible versions.

Reply:
You're right, updated it right now